setup.py

```diff
@@ -7,6 +7,6 @@
       packages=find_packages(exclude=('docs')),
       install_requires=[
           'requests==2.10.0',
-          'pypif==1.0.24',
+          'pypif',
```

Comment:
This is generally not recommended in java/maven world because builds are not necessarily reproducible. Are the conventions different in python?